src/style/toolbar.less

```diff
@@ -35,7 +35,7 @@
 .toolbar {
     box-sizing: border-box;
     z-index: 1;
-    width: @toolbar-button-size;
+    width: @toolbar-button-size + .2rem;
```

Comment:
`.2` here and then `0.2` the line below.  I guess we can let it slide this time ; )